scalafix-reflect/src/main/scala/scalafix/internal/reflect/ScalafixToolbox.scala

```diff
@@ -39,6 +47,14 @@ class ScalafixToolbox {
       code: Input,
       index: LazySemanticdbIndex): Configured[Rule] =
     synchronized {
```

Comment:
Unrelated, but I'm wondering whether getRule itself should be synchronized.

Reply:
Hmm, I'm not sure. I recall this was enough to handle my manual tests running compilation for 20 project concurrently.